test/expr/unary/selector/selector.swift

```diff
@@ -116,7 +116,7 @@ func testParseErrors3(_ c1: C1) {
 
 func testParseErrors4() {
   // Subscripts
-  _ = #selector(C1.subscript) // expected-error{{type 'C1.Type' has no subscript members}}
+  _ = #selector(C1.subscript) // expected-error{{type 'C1' has no member 'subscript'}}
```

Comment:
…and I don't quite understand your objection. How do you want to spell "the selector of the getter for this subscript"?

I suppose we haven't had a huge need for it yet, but it's not an unreasonable thing to ask for.